pkgs/desktops/gnome-3/core/gucharmap/src.nix

```diff
@@ -1,10 +1,10 @@
 # Autogenerated by maintainers/scripts/gnome.sh update
```

Comment:
This comment is obsolete.

Reply:
No.
It is still generated by `./maintainers/scripts/gnome.sh update gucharmap`.
Having `gucharmap` in `NO_GNOME_MAJOR` means that this package is not updated with the rest of GNOME by calling `./maintainers/scripts/gnome.sh update-all`.